packages/errors/src/errors.ts

```diff
@@ -1085,6 +1085,12 @@ export const AllCypressErrors = {
 
         You can safely remove the ${fmt.highlight(`experimentalStudio`)} configuration option from your config.`
   },
+  EXPERIMENTAL_SINGLE_TAB_RUN_MODE: () => {
+    return errTemplate`\
+        The ${fmt.highlight(`experimentalSingleTabRunMode`)} experiment is currently only supported for Component Testing.
+
+        If you are enjoying the experiment with Component Testing, please join the discussion here: http://on.cypress.io/experimental-single-tab-run-mode`
```

Comment:
Maybe have a more open ended message? They may be enjoying it, but they may not be? Either way we would want feedback.

Reply:
https://github.com/cypress-io/cypress/pull/23104/commits/d7aa638467cdc1054541051038285a741387c583